controllers/storagecluster/exporter.go

```diff
@@ -72,6 +72,12 @@ func (r *StorageClusterReconciler) enableMetricsExporter(
 		return err
 	}
 
+	// create/update rook-ceph monitoring rolebindings
+	if err := createRookCephClusterRolebindings(ctx, r, instance); err != nil {
+		r.Log.Error(err, "failed to create/update rolebindings for rook-ceph monitoring component")
```

Comment:
Since we're already logging the error inside that function, we don't need to log it again. We can either remove this log or the one inside that function to reduce noise.

Reply:
Removed